docs/designers-developers/developers/tutorials/block-tutorial/nested-blocks-inner-blocks.md

```diff
@@ -0,0 +1,163 @@
+# Nested Blocks: Using InnerBlocks
+
+You can create a single block that nests other blocks using the [InnerBlocks](/packages/block-editor/src/components/inner-blocks) component. This is used in the Columns block, Social Links block, or any block you want to contain other blocks.
+
+Note: A single block can only contain one `InnerBlock` component.
+
+Here is the basic InnerBlocks usage.
+
+{% codetabs %}
+{% ES5 %}
+```js
+( function( blocks, element, blockEditor ) {
+	var el = element.createElement;
+	var InnerBlocks = blockEditor.InnerBlocks;
+
+	blocks.registerBlockType( 'a8c/workshop-ex7', {
```

Comment:
It looks like every example in the docs refer to existing code in https://github.com/wordpress/gutenberg-examples:

```suggestion
	blocks.registerBlockType( 'gutenberg-examples/example-6', {
```
I guess we need to prepare the accompanying PR in gutenberg examples as well?